editor/modes/visual-editor/block.js

```diff
@@ -70,10 +72,12 @@ function VisualEditorBlock( props ) {
 						isActive: () => control.isActive( block.attributes )
 					} ) ) } />
 			) : null }
-			<BlockEdit
-				isSelected={ isSelected }
-				attributes={ block.attributes }
-				setAttributes={ setAttributes } />
+			<div className="editor-visual-editor__block-edit">
```

Comment:
Hmm would it be possible to effect the padding on the parent `.editor-visual-editor__block` instead of creating a wrapper here?